### PR TITLE
Summary of plans for 2022

### PR DESCRIPTION
**Pull request by graeme-a-stewart**
Based on inputs and discussion in HSF Coordination Meetings early in 2022

### CODE REVIEW

**Comment by graeme-a-stewart** [comment]
N.B. link checker failure is a false positive (known problem with links that are internal to website, see #1046)

File: _includes/navbar.ext [hunk 1]
```diff
@@ -75,7 +75,7 @@
               <ul class="dropdown-menu" aria-labelledby="about_menu">
                 <li><a href="{{ site.baseurl }}/get_involved.html">Get involved!</a><li>
                 <li><a href="{{ site.baseurl }}/organization/team.html">The Coordination Team</a></li>
-                <li><a href="{{ site.baseurl }}/organization/planning/plan-of-work-2021.html">HSF Planning</a></li>
```
Comment (eduardo-rodrigues) [inline _includes/navbar.ext:78]: Out of curiosity - what will then happen with the 2021 planning page? It stays in the site, I see, but then it won't appear anymore anywhere? That would be a shame. We could easily do what we did for the CWP page with a "history" section for previous years, see https://hepsoftwarefoundation.org/organization/cwp.html. Seems appropriate to me.

File: organization/planning/plan-of-work-2022.md [hunk 2]
```diff
@@ -0,0 +1,215 @@
+---
+title: HSF Plans for 2022
+author: Graeme Stewart
+layout: plain_toc
+---
+
+## Preamble
+
+At the beginning of 2022 the HSF established some high-level
+plans and goals for 2022. While none of these are cast in stone, we
```
Comment (eduardo-rodrigues) [inline organization/planning/plan-of-work-2022.md:10]:
```suggestion
plans and goals for the year ahead. While none of these are cast in stone, we
```

File: organization/planning/plan-of-work-2022.md [hunk 3]
```diff
@@ -0,0 +1,215 @@
...
+remain agile to adapt to new opportunities and circumstances throughout
+the year, they are a useful guide to our activities.
```
Comment (eduardo-rodrigues) [inline organization/planning/plan-of-work-2022.md:12]:
```suggestion
the year. This planning is a useful guide to our activities.
```

File: _includes/navbar.ext (hunk 1 above)
Comment (eduardo-rodrigues) [reply _includes/navbar.ext:78]: Ah, answering myself - I now see the link for 2021 below :D.

Comment (graeme-a-stewart) [reply _includes/navbar.ext:78]: Yes, I thought the best is that the planning always points to the current year (so not having to click twice), but with links to previous years always available

File: organization/planning/plan-of-work-2022.md [hunk 4]
```diff
@@ -0,0 +1,215 @@
...
+
+### PyHEP
+
+#### 2022 Workshop
+
+* Since we’re no longer planning to co-locate with SciPy in Austin, TX, choosing a date within a week of SciPy is no longer a benefit
+* We’re considering end of August/beginning of September instead of July.
+* We’re thinking of adding new features (still brainstorming)
+  * Maybe adding hackathon / community building specific contributions
+* Would mean a call for “short tasks” across ecosystem package aside the usual call for presentations
```
Comment (eduardo-rodrigues) [inline organization/planning/plan-of-work-2022.md:121]:
```suggestion
* Would mean a call for “short tasks” across the ecosystem packages aside the usual call for presentations
```

**Review by eduardo-rodrigues — APPROVED** [review]

**Pull request merged**